server/utils/applications/reportDataFromApplication.test.ts

```diff
@@ -336,4 +337,41 @@ describe('reportDataFromApplication', () => {
       expect(isConcerningArsonBehaviourFromApplication(application)).toEqual(undefined)
     })
   })
+
+  describe('releaseTypesFromApplication', () => {
+    it('returns the translated release types', () => {
+      const application = applicationFactory.build({
+        data: {
+          'sentence-information': {
```

Comment:
Just a heads up that in the past I've had issues with these factories not creating the `data` property correctly when adding deeply nested properties. 
Doesn't look to be an issue here but thought it was worth mentioning in case it saves you some head scratching in future.